_episodes/44.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: episode
+title: "44: Seeing clearly with Metabase"
+date: 2023-01-26
+tags: trino metabase visualization client clojure bi reporting
+youtube_id: ""
+wistia_id: ""
+sections:
+   - title: "Release update"
+     desc: "405 and 406"
+     time: 
+---
+
+## Hosts
+
+* Manfred Moser, Director of Information Engineering at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+
+## Guests
+
+* [Luis Paolini](https://www.linkedin.com/in/luispaolini/), Product Advocate at
+  [Metabase](https://www.metabase.com/)
+* [Andrew DiBiasio](https://www.linkedin.com/in/andrewdibiasio/), Software
+  Engineer at [Starburst](https://starburst.io)
+* [Piotr Leniartek](https://www.linkedin.com/in/piotrleniartek), Project Manager
+  at [Starburst](https://starburst.io)
+
+## Recap of Trino in 2022
+
+Highlights from the blog post [The rabbit reflects on Trino in 2022]({% post_url
+2023-01-10-trino-2022-the-rabbit-reflects %}):
+
+* 
+
+## Releases 404 to 406
+
+Official highlights from Martin Traverso:
+
+[Trino 404]({{site.url}}/docs/current/release/release-404.html) not found
+
+[Trino 405]({{site.url}}/docs/current/release/release-405.html)
+
+* Support for `ALTER COLUMN ... SET DATA TYPE` statement.
+* Support for Apache Arrow when reading from BigQuery.
+* Support for views in the Delta Lake connector.
+* Support for the Iceberg REST catalog.
+* Support for Protobuf encoding in the Kafka connector.
+* Support for fault-tolerant execution in the MongoDB connector.
+* Support for `DELETE` and query pushdown in the Redshift connector.
+* Performance improvements when reading Parquet data.
+
+[Trino 406]({{site.url}}/docs/current/release/release-406.html)
+
+* tbd
```

Comment:
Pending 406 coming out on Wednesday or earlier. At min we want to give a status update @colebow

Reply:
```suggestion
* Fault-tolerant execution for the BigQuery connector
* Better fault-tolerant execution with HDFS
* Support for the JDBC catalog in Iceberg
* Many performance improvements for working with Parquet files in connectors
```